src/client/scripts/views/CreateAccount/NeedHousingView.js

```diff
@@ -22,29 +21,123 @@ export class NeedHousingView {
         content.innerHTML = `
             <h2>Create Account - Need Housing</h2>
             <form id="needHousingForm">
-                <label for="firstName">First Name:</label>
-                <input type="text" id="firstName" name="firstName" required><br><br>
-                
-                <label for="lastName">Last Name:</label>
-                <input type="text" id="lastName" name="lastName" required><br><br>
-                
-                <label for="email">Email:</label>
-                <input type="email" id="email" name="email" required><br><br>
-                
-                <label for="phone">Phone:</label>
-                <input type="tel" id="phone" name="phone" required><br><br>
-                
                 <label for="housingPreferences">Housing Preferences:</label>
-                <textarea id="housingPreferences" name="housingPreferences" required></textarea><br><br>
-                
+                <div class="input-box">
```

Comment:
Use TextInput. TextAreaInput, and CheckboxInput components instead of doing it manually

Reply:
Sorry it's nitpicky but it's part of our grade to keep code modular and reusable

Reply:
No worries, I'll go change that right now thank you so much